crates/storage/nippy-jar/src/compression/mod.rs

```diff
@@ -1,17 +1,23 @@
 use crate::NippyJarError;
 use serde::{Deserialize, Serialize};
-use std::io::Write;
 
 mod zstd;
-pub use self::zstd::{Zstd, ZstdState};
+pub use self::zstd::{DecoderDictionary, Decompressor, Zstd, ZstdState};
+mod lz4;
+pub use self::lz4::Lz4;
 
 /// Trait that will compress column values
 pub trait Compression: Serialize + for<'a> Deserialize<'a> {
+    /// Appends decompressed data to the dest buffer. `dest` should be allocated with enough memory.
```

Comment:
this is a requirement, right? 
it must have sufficient capacity to fit the decompressed value

Reply:
yes it's a requirement. changed the wording